src/inject/dynamic-theme/modify-css.ts

```diff
@@ -303,7 +303,7 @@ function shouldIgnoreImage(selectorText: string, selectors: string[]) {
     const ruleSelectors = selectorText.split(/,\s*/g);
     for (let i = 0; i < selectors.length; i++) {
         const ignoredSelector = selectors[i];
-        if (ruleSelectors.some((s) => s === ignoredSelector)) {
+        if (ruleSelectors.some((s) => typeof ignoredSelector === 'string' ? s === ignoredSelector : ignoredSelector.test(s))) {
```

Comment:
I suggest you to use strings and check if the `ignoredSelector` ends with `*` here. If so, make another code branch in `if...else` block.

Reply:
> I suggest you to use strings and check if the `ignoredSelector` ends with `*` here. If so, make another code branch in `if...else` block.

"*" can not handle these:

```
// :: selector
.xxx > section > headerp[v-data-xxxxxxxx] :: before { /* ...  */ }
// deep not scoped selector, like some UI components
.xxx > section > headerp[v-data-xxxxxxxx] > .yyy > .zzz { /* ...  */ }
```

@alexanderby

Reply:
Both selectors in your example start with `.xxx > section > header[data-v-`. The selector is split into parts on line 303. That means that the config can have `header[data-v-*` selector and easily compare the part of the selector with `s.startsWith(ignoredSelector.substring(0, ignoredSelector.length - 1))`.